build_info/gitea.control

```diff
@@ -0,0 +1,9 @@
+Package: gitea
+Version: @DEB_GITEA_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Section: Data_Storage
+Priority: optional
+Depends: git, libpam2
+Pre-Depends: system-cmds (>= 854.40.2-2)
+Description: Git with a cup of tea, painless self-hosted git service
```

Comment:
Add the gitea homepage above the Description field; do this for the macOS control file as well.